plasma_framework/test/src/framework/ExitGameController.test.js

```diff
@@ -144,6 +144,27 @@ contract('ExitGameController', () => {
             expect(await priorityQueue.currentSize()).to.be.bignumber.equal(new BN(2));
         });
 
+        it('emits an ExitEnqueued event', async () => {
```

Comment:
This can be put under the describe below: `'when successully enqueued'` so we can reuse the setup for enqueue.

ah...and just realized there's a typo there on `successully` >"< Do you mind to fix that?